modules/nf-core/falco/main.nf

```diff
@@ -48,8 +45,9 @@ process FALCO {
     stub:
     def prefix = task.ext.prefix ?: "${meta.id}"
     """
-    touch ${prefix}.html
-    touch ${prefix}_fastqc_data.txt
+    touch ${reads}_fastqc_data.html
```

Comment:
This needs to stay as `prefix` in the stub I think

Reply:
yes